wger/core/views/user.py

```diff
@@ -540,3 +547,180 @@ def get_context_data(self, **kwargs):
             context['object_list']['members']
         }
         return context
+def oauthLogin(request, platform):
+    '''
+        Social networks login
+    '''
+    authorize_url = '{}/en/user/login/oauth/facebook/authorize'.format(os.getenv('SITE_URL'))
+    if platform=='facebook':
+        # Redirect to facebook login page
+        url = 'https://www.facebook.com/v3.0/dialog/oauth?client_id={}&redirect_uri={}&state={}&scope={}'.format(
+                '470369423388367',
```

Comment:
this is sensitive data and should not be pushed repo.

Reply:
@parseendavid Thank you for this. Let me remove it.